src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
+      this.result = callback(this.result, value);
+
+      return this;
+    },
+
+    add(x, y) {
+      const result = x + y;
+
+      return result;
+    },
```

Comment:
тут можна нічого не повертати , а лише записувати значення в result
```suggestion
    add(x) {
      calculator.result += x;
    },
```

Reply:
виправила